Marlin/src/module/temperature.cpp

```diff
@@ -849,6 +849,15 @@ void Temperature::min_temp_error(const heater_ind_t heater) {
             }
           #endif // PID_EXTRUSION_SCALING
 
+          #if ENABLED(PID_FAN_SCALING)
+            if (thermalManager.fan_speed[active_extruder] > PID_FAN_SCALING_MIN_SPEED) {
```

Comment:
I'm going to suggest adding work_pid[ee].Kf = 0; here to zero out the Kf value like is done with the Kc value.  I think the main value of this is to show 0 on the pid debug output when the fan isn't on thus making the debug output less confusing.